docker/install-dynamic-plugins.py

```diff
@@ -0,0 +1,84 @@
+import os
+import sys
+import yaml
+import tarfile
+import shutil
+import subprocess
```

Comment:
are any of these deps installed via pip? if so we need a requirements.txt

Examples:

https://github.com/redhat-developer/devspaces-jiralint/blob/devspaces-3-rhel-8/requirements.txt
https://github.com/redhat-developer/devspaces-images/tree/devspaces-3-rhel-8/devspaces-udi/build/python

Reply:
Yes if I remove the installation of `mkdocs-techdocs-core` by pip, the `yaml` package which we need in the dynamic-plugins-install.py script is not available anymore.
So I assume we need a requirements.txt file.

Reply:
I added a `requirements.txt` file is this commit, but I have no idea if it is the place to put it.

Reply:
next to the .py script it refers to is fine. Once this is merged I can start testing it downstream ... or now that 0.2 is done, I can start buildin RHDH from this fork/branch, if you want to get some input on how the python cachito stuff works.